databricks_sql_connector_core/tests/e2e/test_complex_types.py

```diff
@@ -4,6 +4,9 @@
 from tests.e2e.test_driver import PySQLPytestTestCase
 
 
+@pytest.mark.skipif(
+    pytest.importorskip("pyarrow"), reason="Skipping because pyarrow is not installed"
+)
```

Comment:
Okay, so I've seen at least three ways to conditionally skip tests when `pyarrow` not available 🙂 Please choose one of them and stick with it everywhere.

Also, please note that here `pytest.importskip` is not used correctly (see https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-importorskip). It should be used in a test body to skip the remaining part of it. `@pytest.mark.skipif` decorator seems to be a better fit if we want to skip the whole test case, but in this case it's better to use it with the predicate you created to check the presense of `pyarrow`

Reply:
fixed

Reply:
fixed